### PR TITLE
feat: add support for decompress plugin

### PR DESCRIPTION
**Pull request by mastilver**

### CODE REVIEW

**Comment by mastilver** [comment]
plugin can be set on the `opts` object
Is it good that way?

Or would it be better to add a new arguments?

File: readme.md [hunk 1]
```diff
@@ -38,6 +38,12 @@ Type: `number`
 
 Equivalent to `--strip-components` for tar.
 
+### use
+
+Type: `Array`, `Function`
```
Comment (sindresorhus) [inline readme.md]: lowercase types

File: readme.md [hunk 2]
```diff
@@ -38,6 +38,12 @@ Type: `number`
...
+
+[decompress](https://github.com/kevva/decompress) plugin to use
```
Comment (sindresorhus) [inline readme.md]: end it with a dot `.`

**Comment by kevva** [comment]
Fixed in https://github.com/kevva/gulp-decompress/commit/fc802c8d1b59ca5de2ca160b09c665fd38cb38f0. This module now accepts whatever options that `decompress` accepts.

**Comment by mastilver** [comment]
billant, thank you!

**Pull request closed without merging**